### ISSUE TITLE
Customization of chat message if hacker is identified on opposite team

### ISSUE THREAD

**Issue by DiegoFleitas** [PazerOP/tf2_bot_detector#70]
Either that or straight away disabling this would be my pick. Cheaters keep logs of chat so this message is way for them to single out players who are running this tool which most people don't feel comfortable with

**Comment by Scarlette564** [comment]
This is not an issue, please only put issues here.

**Comment by Scarlette564** [comment]
Use "Pull requests"

**Comment by DiegoFleitas** [comment]
I would use the label enhancement, I'm not proficient enough with C++ to provide a solution

**Comment by ClusterConsultant** [comment]
This is partially solved in recent builds. Currently you can disable all chat warnings. If you wish you can also change the messages and then compile the tool yourself. Future releases may have a more user friendly method.

**Comment by ClusterConsultant** [comment]
This is _the_ most commonly requested feature. It will be done at some point.

Right now there are technical problems with implementing any kind of user customizable message but work is being done on it. If you are desperate you can change the messages and compile the tool yourself but this will break functionality and is only recommended for people who already know what they are doing. 

This is being worked on.

**Comment by id-rotatcepS** [comment]
if it includes tag substitution in the message, it could get pretty fun.  
My favorite line to use (when they were common) was:
"Oh myg0t, Becky, look at BLU's bot!﻿‏‬‎"

**Comment by brachistosaurus** [comment]
Instead of repeating the same message, it could cycle through (or randomly choose from) a list of user-defined templates. This may benefit users who want the automated messages to seem less "automated".

Template examples:

- Guys, there are 3 cheaters on the enemy team (Bot1, Bot2, and Bot3).
- 3 cheaters are here: Bot1, Bot2, and Bot3
- Can the other team kick Bot1, Bot2, and Bot3?
- please KICK Bot1, Bot2, and Bot3.
- Bot1, Bot2, Bot3 are hackers/bots

**Comment by brandons209** [comment]
Yeah, I would say this needs to be a higher priority. Currently, the main issue is with cheaters who are real players. They typically will convince others the you are a bot/hacker because of the automated message that doesn't seem more... unique or natural. Recently, I also witnessed bots/cheaters now using the default chat messages to pretend they are using the bot detector and call out random people, thus getting them kicked or people identifying this automated message as a bot account and not a person using the bot detector. With this is usually chat spam and other bot related activities, which seems to be trying to put a bad name to those using the bot detector. Personally, cheaters are usually really hard to get with this unless they are just joining on your team.

**Comment by TheDrawingCoder-Gamer** [comment]
I also see this as very important, as there are many bots/cheat tools that clear chat when it sees a word that relates to bots. Being able to customize it so it isn't as "on the nose" or not as automatable to clear chat.

**Comment by imaginebeingatcomputers** [comment]
A slight danger with using custom warnings would be it could lead to spam in the case of multiple players are running TF2BD on the same team. Bot devs would also really only have to detect if the name of their bot is being referenced by another player to determine who to target.

**Comment by Tiagoquix** [comment]
> This is being worked on.

@ClusterConsultant Considering your comment was made on 2021-01-15, and the latest release was made available to the public on 2021-06-26 without having this added, what is the current status of the implementation?
